cds-snc/cppd-medical-report-sails
# Doctor has a confirmation page after submitting the report

# What this does
Updates the confirmation screen when a doctor submits a report [according to the design described here](https://trello.com/c/77QgwYd6/539-doctor-has-a-confirmation-page-after-submitting-the-report).

NOTE: This does not include the new *Your information* or *Declaration/Review* screens

## To test
- Create a medical report
- Fill out all the sections
- Submit the report
- Notice the new Confirmation screen
